bigtop-packages/src/common/alluxio/install_alluxio.sh

```diff
@@ -65,12 +65,12 @@ while true ; do
         --bin-dir)
         BIN_DIR=$2 ; shift 2
         ;;
-        --var-dir)
-        VAR_DIR=$2 ; shift 2
-        ;;
```

Comment:
[bigtop-packages/src/deb/alluxio/rules](https://github.com/apache/bigtop/blob/0f67454a7b71f54b6098ef44cb459e656853a0d7/bigtop-packages/src/deb/alluxio/rules#L39) must be updated if you remove the option.

```
$ ./gradlew alluxio-pkg
...
bash -x debian/install_alluxio.sh \
    --build-dir=build \
    --bin-dir=/usr/bin \
    --data-dir=/usr/share \
    --libexec-dir=/usr/lib/alluxio/libexec \
    --var-dir=/var \
    --prefix=debian/tmp
+ set -ex
++ getopt -n debian/install_alluxio.sh -o '' -l prefix: -l bin-dir: -l libexec-dir: -l lib-dir: -l data-dir: -l conf-dist-dir: -l build-dir: -- --build-dir=build --bin-dir=/usr/bin --data-dir=/usr/share --libexec-dir=/usr/lib/alluxio/libexec --var-dir=/var --prefix=debian/tmp
debian/install_alluxio.sh: unrecognized option '--var-dir=/var'
+ OPTS=' --build-dir '\''build'\'' --bin-dir '\''/usr/bin'\'' --data-dir '\''/usr/share'\'' --libexec-dir '\''/usr/lib/alluxio/libexec'\'' --prefix '\''debian/tmp'\'' --'
make[1]: *** [debian/rules:34: override_dh_auto_install] Error 1
make[1]: Leaving directory '/home/iwasakims/srcs/bigtop/output/alluxio/alluxio-2.9.3'
make: *** [debian/rules:27: binary] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -us -uc -ui -b
 failed

> Task :alluxio-deb FAILED
```